docs/source/api_ref_dtypes.rst

```diff
@@ -22,12 +22,10 @@ Layouts and Tensor Subclasses
     FloatxTensor
     FloatxTensorCoreLayout
     MarlinSparseLayout
-    BlockSparseLayout
```

Comment:
looks like it's both? maybe try to split would be better, or update the summary to say it's two things